src/Appwrite/Auth/Authentication.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Appwrite\Auth;
+
+class Authentication
+{
+    /**
+     * User Unique ID.
+     *
+     * @var string
+     */
+    private $unique = '';
+
```

Comment:
what is the function of this class? Why did we remove it from Auth? The name Authentication doesnt reflect what this class does?

Reply:
This class holds the user's Cookie and secret.
Before that the details were part of the Auth class and they were static. meaning if two users are accessing the server in the same worker using different coroutines, they will share those details

Reply:
We probably need a better name to reflect its purpose